app/models/gobierto_calendars/event.rb

```diff
@@ -63,6 +63,16 @@ class Event < ApplicationRecord
 
     enum state: { pending: 0, published: 1 }
 
+    def self.events_in_collections_and_container_type(site, container_type)
+      ids = GobiertoCommon::CollectionItem.where(item_type: "GobiertoCalendars::Event", container_type: container_type).map(&:item_id)
```

Comment:
We prefer pluck vs map, because pluck doesn't loads the whole object and is much faster. See this post: https://rubyinrails.com/2014/06/05/rails-pluck-vs-select-map-collect/

Reply:
https://github.com/PopulateTools/gobierto/pull/948/commits/dde9c136fc3e6df9c23b681940c259467084e6cb